apps/react-workshop/src/Select.stories.tsx

```diff
@@ -157,7 +157,10 @@ export const Custom = () => {
     <Select
       options={options}
       value={selectedValue}
-      onChange={setSelectedValue}
+      onChange={React.useCallback(
+        (value: string) => setSelectedValue(value),
```

Comment:
The explicit type annotation is usually unnecessary for params. Does it not get automatically inferred?

```suggestion
        (value) => setSelectedValue(value),
```